optimade/server/query_params.py

```diff
@@ -213,7 +213,7 @@ def __init__(
             ge=0,
         ),
         page_number: int = Query(
-            0,
+            1,
```

Comment:
I used the value of 0 to indicate that the page_number parameter is unset.
This allows throwing an error when more than one pagination method is used.
A quick fix would be to set the default to "None".  
```suggestion
            None,
```
I am however, not sure what the consequence of this is for the OpenAPI Schema.